### ISSUE TITLE
Filtering media display by layer

### ISSUE THREAD

**Issue by cmcneave** [GCRC/nunaliit#419]
We have an atlas that shares geographic features across multiple modules, and some of these features have media (audio, video, photos, and text) that are module-specific.  For data quality and maintenance we do not want to duplicate these geographic features, but because the modules are about different topics, their media files should display only on specific modules.

For consistency, we would like to have the ability to filter the display of media documents in a given module by layer.  This is similar to Nunaliit's ability to filter search results by module.  If a layer property contained in a media document is included in the module, the it will be shown as a related document.

**Comment by jpfiset** [comment]
Basically, adding a filter to the documents reported in the related section of the display?

**Comment by cmcneave** [comment]
Yes, that would be great.  This sounds like when adding related documents a choice of possible modules is offered...?

**Comment by jpfiset** [comment]
I was thinking along the lines of adding code in the nunaliit_custom.js file to set up a filter with the display logic. This way, we could filter to the current modules the documents displayed.

**Comment by pulsifer** [comment]
This seems correct.  Ideally the filtering system would be quite flexible, but using nunaliit_layers (which in Nunaliit are more comparable to a class, I think) would be a great start.  So, the script would say that Module A should display only documents where nunaliit_layers contains X,Y,Z.  Or, Module A should display all documents except those where nunaliit_layers contains W.  Although we will often being filterin g out documents without geometries (e.g. media documents), as indicated above, we will use a nunaliit_layers array in those documents to control the filtering.  Ideally, we could also define filter based on other attributes as well.
